services/telegram-bot/pod-telegram-bot/src/account.ts

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import { ClientWorkspaceInfo } from '@hcengineering/account'
+
+import config from './config'
+
+export async function getWorkspaceInfo (token: string): Promise<ClientWorkspaceInfo | undefined> {
```

Comment:
we have client package to access account, please use it instead.

Reply:
Done